corehq/form_processor/migrations/sql_templates/migrate_balance_3_backfill_notnull.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE form_processor_ledgertransaction SET new_updated_balance = updated_balance;
```

Comment:
Would it make sense to add `WHERE new_updated_balance IS NULL` to ignore rows already updated by the trigger? (assuming the trigger is required, if not might be simpler to eliminate the trigger instead)

Reply:
+1 to adding the filter